tls/s2n_connection.c

```diff
@@ -1517,3 +1518,41 @@ S2N_RESULT s2n_connection_dynamic_free_in_buffer(struct s2n_connection *conn)
 
     return S2N_RESULT_OK;
 }
+
+/* Marks s2n_ktls_mode enabled for the connection.
+ *
+ * Note: currently, kTLS cannot be disabled once enabled.
+ */
+S2N_RESULT s2n_connection_mark_ktls_enabled(struct s2n_connection *conn, s2n_ktls_mode ktls_mode)
```

Comment:
I am confused. Are both of these functions that you've added APIs? Why is only one declared in connection.h?